x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.scss

```diff
@@ -27,13 +28,26 @@
   margin-left: $euiSizeXS / 2;
   margin-bottom: $euiSizeXS / 2;
 
+  &:focus {
+    @include euiFocusRing;
+    transform: none !important; // sass-lint:disable-line no-important
+  }
+
   .lnsSuggestionPanel__expressionRenderer {
     position: static; // Let the progress indicator position itself against the button
   }
 }
 
 .lnsSuggestionPanel__button-isSelected {
```

Comment:
Please add a `border-color: $euiColorMediumShade;` within this selector's styles. I think this added contrast will help the selected button stand out just a bit more.